app.js

```diff
@@ -90,10 +90,23 @@ app.get('/delete', function(req, res) {
   // TODO add your code here...
 
 });
+
 app.get('/', function(req, res) {
   res.sendfile(__dirname + '/index.html');
 });
 
+app.get('/index', function(req, res) {
```

Comment:
Not needed. This adds the endpoint "/index" to the Node server, in addition to the endpoint "/". Both go to the same place. Please remove.